plugins/kafka/kafka.cpp

```diff
@@ -506,8 +526,17 @@ namespace KafkaPlugin
             consumerPtr->stop(topicPtr, partitionNum);
         }
 
-        delete(topicPtr.load());
-        delete(consumerPtr);
+        if (topicPtr)
```

Comment:
Why this change?  You don't need to protect delete of a null pointer, and this is within the destructor so the values will never be accessed.

Reply:
Will revert.